go.mod

```
@@ -1,6 +1,6 @@
 module github.com/elastic/integrations
 
-go 1.19
+go 1.21
```

Use 1.21.0 to avoid the issues we have seen in other repos.

```suggestion
go 1.21.0
```

---

Right
I used `go mod edit -go=1.21` and that was the update done by that command

Trying to run `go mod edit -go=1.21.0` failed.